### DISCUSSION TITLE
I'm signed in to Copilot in VS2022 (green checkmark) but it won't do anything.

### DISCUSSION THREAD

**Discussion by angusgraham** [community/community#137198] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/user-attachments/assets/ebc3de64-0ee1-41cb-b2c5-dbd9a3f65555)

Above is an example of the problem. As you can see I'm signed in to github copilot.
When I type anything in the chat window it sticks on "Thinking..." for a few minutes and then gives up and says "Hi! Let's build together."
I'd love to, thanks.
Also, when typing in the code window, copilot doesn't respond at all.
The advice at https://docs.github.com/en/copilot/quickstart is to use Help | Send Feedback | Report a Problem, but when I do I just see this:
![image](https://github.com/user-attachments/assets/9fc79450-909d-4157-a40c-4e5b0aa923ac)

...which is maybe related.

But without "Report a Problem" working I'm stuck which is why I'm posting here and hoping you can help me with this.

**Comment by ayim** [comment]
@angusgraham sorry you're experiencing this, could you use the Report a Problem feature in Visual Studio so that we can have a few more logs to diagnose with?
![image](https://github.com/user-attachments/assets/c2deaf99-441b-4893-b5d3-007aab385fb7)


Side note, what was the first query you tried to send with GitHub Copilot Chat? I'm curious :)

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio

**Reply by angusgraham** [reply]
Hi Aaron,
I tried selecting Report a Problem from the top-right icon menu instead of from the Help | Send Feedback menu as you suggest but it does the exact same thing.

**Reply by ayim** [reply]
@angusgraham do you have a link to the "Report a Problem" or Send Feedback issue that you would've created through that link?

**Comment by angusgraham** [comment, marked as answer]
Hi Aaron,

So as it turns out, because I was using my personal copilot license and not a company-assigned one, IT had not holed the firewall to let copilot do its thing. Once I went through the proper steps IT waved the wand to make it work.
So not necessarily a bug - tho I might suggest that in cases like this if the copilot plugin could pop an error dialog to pass on the fact that it's broken because of network error instead of just stalling out and include some details about what the error was it would help people debug this case.

The "Report a Problem" thing is still broken by the way, so if I ever do need to use that I will have to get IT to figure out what port THAT'S communicating on and let me through - again it's tough without any readable error messages tho.

**Reply by ayim** [reply]
That's a fair point, View > Output > GitHub Copilot often has more details there

**Comment by angusgraham** [comment]
For the record, my company's IT has also fixed the issue of "Report a Problem" not working. It was also some firewall thing: "the URL was doing some redirects and our SSL Decrypt was interfering with it."

**Reply by ayim** [reply]
That's really helpful, I've just passed this onto our feedback team!
